users/src/main/resources/application.yml

```diff
@@ -1,38 +1,31 @@
-server:
-  port: 8082
-
-spring:
-  application:
-    name: user-service
-  config:
-    import: optional:configserver:${CONFIG_SERVER_URI:http://localhost:8888}
-  # User Service Database
-  datasource:
-    url: jdbc:postgresql://localhost:5432/users
-    username: postgres
-    password: postgres
-
-  # JPA specific for user-service
-  jpa:
-    hibernate:
-      ddl-auto: update
-    show-sql: true
-
-  # OAuth2 Resource Server
-  security:
-    oauth2:
-      resourceserver:
-        jwt:
-          issuer-uri: http://localhost:8080/realms/learning-platform
-          jwk-set-uri: http://localhost:8080/realms/learning-platform/protocol/openid-connect/certs
-
-eureka:
-  instance:
-    instance-id: ${spring.application.name}:${random.value}
-    metadata-map:
-      zone: zone1
-
-logging:
-  level:
-    com.example.userservice: DEBUG
+server:
+  port: 8082
+
+spring:
+  application:
+    name: user-service
+  config:
+    import: optional:configserver:${CONFIG_SERVER_URI:http://localhost:8888}
+  # User Service Database
+  datasource:
+    url: jdbc:postgresql://localhost:5432/users
+    username: postgres
+    password: postgres
+
+  # JPA specific for user-service
+  jpa:
+    hibernate:
+      ddl-auto: update
+    show-sql: true
+
+
+eureka:
+  instance:
+    instance-id: ${spring.application.name}:${random.value}
+    metadata-map:
+      zone: zone1
+
+logging:
+  level:
+    com.example.userservice: DEBUG

```

Comment:
DEBUG level bi trebalo setovati samo za lokalni development. Info da bude defoltno. 
To mozes izmjeniti kad napravis konfiguracione fajlove za razlicite ENV-ove